src/json_printer.sql

```diff
@@ -55,40 +70,65 @@ package body "JSON_PRINTER" as
       return str;
     end if;
   end llcheck;  
-
-  function escapeString(str varchar2) return varchar2 as
-    sb varchar2(32767) := '';
-    buf varchar2(40);
-    num number;
+  
+  -- escapes a single character. 
+  function escapeChar(ch char) return varchar2 deterministic is
```

Comment:
Opps i wanted to edit my previous comment removing my last statement (which actually was wrong,  since there was no escapechar function in the original code...),  not to add a new one... 
Sorry.  I am new to github...